test/es6module/multiple-roots-circular.js

```diff
@@ -5,8 +5,6 @@
 
 // Tests that circular overlapping module dependencies are all loaded before execution
 
-WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
```

Comment:
This change isn't really related - was reading over test cases to pick what to test the -module flag with and noticed this mistake from a previous commit of mine so removed it.